daffodil-cli/src/it/scala/org/apache/daffodil/xml/TestXMLConversionControl.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.xml
+
+import org.apache.commons.io.FileUtils
+import org.junit.Test
+import org.apache.daffodil.CLI.Util._
+import org.apache.daffodil.Main.ExitCode
+import org.junit.Assert.assertTrue
+
+import java.nio.charset.StandardCharsets
+
+class TestXMLConversionControl {
+
+  //
+  // To run tests conveniently under IntelliJ IDEA,
+  // rename the src/test dir to src/test1. Rename the src/it dir to src/test.
+  // Then modify this val to be "test".
+  // Then you can run these as ordinary junit-style tests under the IDE.
+  val test = "it"
```

Comment:
I'm of the opinion that the 267 integration tests run fast enough now that we should just lump them in under regular src/test now, and not bother with the separate src/it.

However, it would be good to move the stage build tasks so that you don't have to wait for the javadoc/scaladoc to build every time you want to make a small changes and then rerun the tests.

Reply:
Created https://issues.apache.org/jira/browse/DAFFODIL-2772 about fixing the src/it tests to be ordinary src/test.